### ISSUE TITLE
Standalone EC2 Templates Fail to Define the "reboot" Comman for `cfn-init`

### ISSUE THREAD

**Issue by ferricoxide** [plus3it/cfn-jenkins#13]
### Problem Description:

If the `NoReboot` parameter is fed a value of `false`, deployment will fail because the `reboot` command referenced by `cfn-init` is not actually defined

### Expected Behavior:

User _should_ be able to feed either boolean-value to the `NoReboot` parameter.

### Actual Behavior:

As written, only the `true` value is supported when feeding booleans to the `NoReboot` parameter.

### (Detailed) Steps to reproduce:

Launch template with `NoReboot` set to `false`

### (Optional) Fix recommendation:

Add a:

~~~
"reboot": {
  "commands": {
    "10-reboot": {
      "command": "shutdown -r +1 &"
    }
  }
},
~~~

Block to the `"AWS::CloudFormation::Init": {}` section of the [`Templates/make_jenkins_EC2-instance.tmplt.json`](/plus3it/cfn-jenkins/blob/master/Templates/make_jenkins_EC2-instance.tmplt.json) and [`Templates/make_jenkins_Agent-instance.tmplt.json`](/plus3it/cfn-jenkins/blob/master/Templates/make_jenkins_Agent-instance.tmplt.json) files.

**Comment by ferricoxide** [comment]
@murphyj21 dunno where you want this on the priority list &mdash; first it's been triggered since the templates were authored (May 2017!).

**Comment by ferricoxide** [comment]
Closed by #19
